Sources/FluidAudio/ASR/TdtDecoder.swift

```diff
@@ -8,25 +8,22 @@
 import CoreML
 import Foundation
 import OSLog
+import Accelerate
 
```

Comment:
https://github.com/FluidInference/FluidAudio/pull/41/files

Reverted my changes here, the TD coreML model doesn't do much, removed it and RTFx is still ~69

Reply:
I dont think we'll have merge conflicts this way